aten/src/ATen/function_wrapper.py

```diff
@@ -131,8 +131,12 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 # add non-virtual declaration to Tensor.cpp
 TENSOR_METHOD_DEFINITION = CodeTemplate("""\
 inline ${return_type} Tensor::${api_name}(${method_formals}) const {
+#ifdef USE_STATIC_DISPATCH
+    ${mobile_method_body}
```

Comment:
nit: you might want to call this "static_dispatch_method_body" to be consistent with the macro name?